src/Buttons/index.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable flowtype/require-valid-file-annotation */
+
+import { IconButton } from './IconButton'
+import { PrimaryButton } from './PrimaryButton.js'
+import { SecondaryButton } from './SecondaryButton.js'
+import { TertiaryButton } from './TertiaryButton.js'
+import { TextAndIconButton } from './TextAndIconButton'
+import { TextButton } from './TextButton'
+
+export { PrimaryButton, SecondaryButton, TertiaryButton, IconButton, TextAndIconButton, TextButton }
+export const Buttons = { PrimaryButton, SecondaryButton, TertiaryButton, IconButton, TextAndIconButton, TextButton }
```

Comment:
I don't like seeing 'index.js' files like this, since they just make it harder to read the code. Even worse, this file exports the same stuff twice. If you are going to do this, please just pick one way (the first one) and stick with it.